### PR TITLE
Milestone4

### PR DESCRIPTION
**Pull request by merclehoczky**
Milestone 4 is ready to be launched. Please review if you agree.

### CODE REVIEW

File: Dockerfile_old
Comment (ViktoriiaTan) [inline Dockerfile_old:1]: maybe we don't need this file anymore?

File: README.md [hunk 1]
```diff
@@ -47,11 +47,24 @@ If not installed, follow the instructions from [Docker's official website](https
...
+3. Create a '.env' file in the same folder (DSToolkits) using your API key in the following format: `WANDB_API_KEY=your_api_key_here`
+4. Run `docker build -t mnist_classifier . `  
+5. Run `docker run --env-file .env mnist_classifier`
+
+------------------------------------------------------------------------------------
+
+**Dockerized environment:**
 
 1. Navigate to the project directory:`cd DSToolkits`   
 2. Build and run the Docker images: `docker-compose up --build`
```
Comment (ViktoriiaTan) [inline README.md]: We didn't use docker compose in this milestone , so can be removed as well?

**Review by ViktoriiaTan — CHANGES_REQUESTED** [review]
We should remove few unnecessary things.Check my comments

File: script/1_main (copy).py
Comment (merclehoczky) [inline script/1_main (copy).py:1]: Is this script necessary?

File: Dockerfile_old
Comment (merclehoczky) [reply Dockerfile_old:1]: You are probably right.

File: README.md (hunk 1 above)
Comment (merclehoczky) [reply README.md]: Honestly, this one I would keep in until we finish Milestone 5 and finalize the ReadMe.

**Pull request merged**